lib/oneview-sdk/cli.rb

```diff
@@ -114,7 +114,7 @@ def version
     # Show environment variables for oneview-sdk-ruby
     def env
       data = {}
-      OneviewSDK::ENV_VARS.each { |k| data[k] = ENV[k] unless k.include?('I3S') }
+      OneviewSDK::ENV_VARS.each { |k| data[k] = ENV[k] }
```

Comment:
I also updated the CLI to output the I3S environment variables. The CLI doesn't necessarily use them yet, but the I3S client does